packages/ui/src/components/Search/FLAutoComplete.tsx

```diff
@@ -0,0 +1,76 @@
+import React, { useState } from 'react';
+import { AutoComplete, Select } from 'antd';
+import { debounce } from 'lodash';
+
+import styles from './fl-auto-complete.module.css';
+
+const DEFAULT_DEBOUNCE_INTERVAL = 250;
+
+export interface FLAutoCompleteOption {
+    id: string;
+    highlight: string;
+    data?: any;
+}
+
+export interface FLAutoCompleteProps {
```

Comment:
juste pour comprendre pourquoi FL au début ?

Reply:
Pour ferlab, je me dis que ca identifie les composant plus rapidement et ca différencie du Autocomplete de Ant

Reply:
ok moi je l'aurai plus fait dans l'import de l'autocomplete dans le portail mais ca se discute. C'est pas une pratique que l'on a mais why not.